### PR TITLE
add docker file to frontend

### PR DESCRIPTION
**Pull request by nonorze-VOXX**

### CODE REVIEW

**Review by NOOBDY — CHANGES_REQUESTED** [review]
.

File: frontend/Dockerfile
Comment (NOOBDY) [inline frontend/Dockerfile:1]: `package-lock.json` is also required

**Review by NOOBDY — CHANGES_REQUESTED** [review]

File: frontend/Dockerfile
Comment (nonorze-VOXX) [reply frontend/Dockerfile:1]: fixed

**Comment by nonorze-VOXX** [comment]
copy .example.env to .env for run

**Review by huhuhu0420 — DISMISSED** [review]

**Review by wj461 — APPROVED** [review]

**Pull request merged**